app/Repositories/PhotoRepository.php

```diff
@@ -154,12 +155,44 @@ public function reviews($data)
         foreach ($data as $review) {
             if (isset($review['rogue_event_id']) && ! empty($review['rogue_event_id'])) {
                 $post = Post::where(['event_id' => $review['rogue_event_id']])->first();
+                $photo = Photo::where(['id' => $post->postable_id])->first();
 
                 if ($review['status'] && ! empty($review['status'])) {
                     // @TODO: update to add more details in the event e.g. admin who reviewed, admin's northstar id, etc.
                     $review['submission_type'] = 'admin';
 
-                    Event::create($review);
+                    // Create the Event.
+                    $event = Event::create([
+                        'signup_id' => $post->signup_id,
+                        // Do we want the user's northstar id or the admin's?
+                        'northstar_id' => $post->northstar_id,
```

Comment:
@sbsmith86 @katiecrane @DFurnes do we want the user's northstar id or the admin's? Right now, this is the user's but I'm thinking maybe it makes more sense to have the admin's?